correios/models/user.py

```diff
@@ -224,7 +226,7 @@ def is_declared_value(self):
     def get(cls, number: Union['ExtraService', int]) -> 'ExtraService':
         if isinstance(number, cls):
             return number
-        return cls(number=number, **EXTRA_SERVICES[number])
+        return cls(number=number, **EXTRA_SERVICES[number])  # type: ignore
```

Comment:
Is this comment necessary?

Reply:
Yes, it is @gfabricio. Without mypy gives an weird error saying that `EXTRA_SERVICES[number]` is not a mapping, it is an object. That's happens when I added `'display_on_label': False` on on EXTRA_SERVICES dict.

Reply:
Maybe because False isn't a string?

Reply:
Or, because not all elements are equals (has same length and attributes).

Reply:
@cacarrara, I can add the `'display_on_label'` in all dicts and change
this
```
@classmethod
    def get(cls, number: Union['ExtraService', int]) -> 'ExtraService':
        if isinstance(number, cls):
            return number
        return cls(number=number, **EXTRA_SERVICES[number])  # type: ignore
```
to this
```
@classmethod
    def get(cls, number: Union['ExtraService', int]) -> 'ExtraService':
        if isinstance(number, cls):
            return number
        attrs = EXTRA_SERVICES[number]  # type: Dict[str, Any]
        return cls(number=number, **attrs)
```
Do you think it is better?

Reply:
Well, maybe we can go with this purpose @marcelobelli